usr/share/rear/conf/default.conf

```diff
@@ -3376,15 +3376,34 @@ ELILO_BIN=
 
 ################ ---- custom scripts
 #
-# NOTE: The scripts can be defined as an array to better handly spaces in parameters.
+# NOTE: The scripts can be defined as an array to better handle spaces in parameters.
```

Comment:
it might be better to just delete the comment above, because using an array does not seem to provide any actual advantage in handling spaces in arguments the way how it is done, or does it?